update-bot/wlmbots/lib/template_checker.py

```diff
@@ -11,6 +11,7 @@ def __init__(self, config=None):
             config = {}
         self.tpl_match_regex = None
         self.config = config
+        self._config = config
```

Comment:
Sorry if I will say a complete rubbish here (and sorry that it does not directly relate to pylint checks) but how do these two (I mean `self.config` and `self._config`) actually differ? I have only found `self._config` being set in `config` setter and returned in `config` getter. Is this used for something else? If so, how about some other name for one (or both) of these two fields?

Another thing: how about changing the default value of `config` parameter of the constructor to `{}`? This seems to be the actual default value, not a `None`.